WooCommerce/Classes/ViewRelated/ReusableViews/ValueOneTableViewCell.swift

```diff
@@ -2,17 +2,39 @@ import UIKit
 
 class ValueOneTableViewCell: UITableViewCell {
 
+    /// The style options for the `ValueOneTableViewCell`
+    ///
+    enum Style {
+        /// Uses BodyStyle for the Text  and Subheadline for the DetailText.  This is the default.
+        case primary
+        /// The same as `primary` except that for the DetailText uses a Tertiary text color.
+        case secondary
+    }
+
     override func awakeFromNib() {
         super.awakeFromNib()
 
-        configureBackground()
         configureTextLabel()
         configureDetailTextLabel()
+        apply(style: .primary)
```

Comment:
I think I missed this from an earlier review, why do we need to configure the cell on `awakeFromNib`?

Reply:
Currently the `ValueOneTableViewCell` is used directly without the VM I just added, so the excepted default theme needs to be applied when we are not configuring it using a VM.